frontend/packages/ux-editor/src/components/Settings/SettingsUtils.tsx

```diff
@@ -58,3 +64,27 @@ export const getTaskName = (task: TaskNavigationGroup, layoutSetsModel: LayoutSe
   );
   return matchingTask?.id ?? '-';
 };
+
+type GetHiddenTasksProps = {
+  taskNavigationGroups: TaskNavigationGroup[];
+  layoutSetsModel: LayoutSetsModel;
+};
+
+export const getHiddenTasks = ({ taskNavigationGroups, layoutSetsModel }: GetHiddenTasksProps) => {
```

Comment:
This is missing an explicit return type, maybe `TaskNavigationGroup` from our dto type definitions is suitable here?

```suggestion
export const getHiddenTasks = ({ taskNavigationGroups, layoutSetsModel }: GetHiddenTasksProps): TaskNavigationGroup[] => {
```

Reply:
Thx! Do you know if we will add a linting for this in the future? I remember we have talked about it.

Reply:
Definitely something we should look into, if we can find a way to lint only new code 👍